platform/source/tools/clustercli/common.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015-2017 Applatix, Inc. All rights reserved.
+#
+
+import re
+
+from anytree import Node
+from .cli import AbstractPrompt
+from ax.cluster_management.app.options.common import ClusterOperationDefaults
```

Comment:
Please export these variables (ClusterOperationDefaults, ClusterInstallDefaults) at package level, as `.common`, `.install_options` are internals to package `ax.cluster_management.app.options`, which callers don't need to care